### PR TITLE
fix: changes in UI of password text input in e-CAS modal

### PR DESCRIPTION
**Pull request by akpatel2131**
the eye icons are added to see the password in text input box and modified the height of textIput box

### CODE REVIEW

**Review by tiwariav — CHANGES_REQUESTED** [review]

File: src/components/atoms/FileInput/FileInput.tsx [hunk 1]
```diff
@@ -232,7 +234,25 @@ export default function FileInput<TFile extends UploadFile = UploadFile>({
                       label={dataItem.label}
                       placeholder={dataItem.placeholder}
                       size="small"
-                      type={dataItem.type || "text"}
+                      style={{ paddingTop: 12 }}
```
[the remark this answers is not in the document]
Comment (akpatel2131) [reply src/components/atoms/FileInput/FileInput.tsx:237]: in textInput.tsx file there is paddingTop inline styling. to override that property, we have to pass it in inline style.

File: src/components/atoms/FileInput/FileInput.tsx [hunk 2]
```diff
@@ -222,18 +223,19 @@ export default function FileInput<TFile extends UploadFile = UploadFile>({
...
-                      defaultValue={dataItem.value}
-                      innerClassNames={{ input: styles.listItemDataInput }}
-                      key={index}
-                      label={dataItem.label}
-                      placeholder={dataItem.placeholder}
-                      size="small"
-                      type={dataItem.type || "text"}
-                    />
+                    <>
+                      <div>{dataItem.label}</div>
```
Comment (tiwariav) [inline src/components/atoms/FileInput/FileInput.tsx:227]: why are we not passing the label through `label` prop?

**Review by tiwariav — CHANGES_REQUESTED** [review]

File: src/components/atoms/FileInput/FileInput.tsx (hunk 2 above)
Comment (akpatel2131) [reply src/components/atoms/FileInput/FileInput.tsx:227]: when we pass prop as label. the whole label gets black background color which is not looking nice. that's qhy i removed label from there and add separate label.

**Comment by tiwariav** [comment]
Closing as already update in main

**Pull request closed without merging**